src/tests/test_index_btree.cpp

```diff
@@ -46,7 +46,7 @@ chunk_selector_type_t test_common::HSTestHelper::s_ds_chunk_sel_type;
 
 SISL_OPTION_GROUP(test_index_btree,
                   (num_iters, "", "num_iters", "number of iterations for rand ops",
-                   ::cxxopts::value< uint32_t >()->default_value("65536"), "number"),
+                   ::cxxopts::value< uint32_t >()->default_value("1500"), "number"),
```

Comment:
with 4k , 65k entries should work right ?

Reply:
correct. considering 4 btree node variations, this is for building purpose to make it fast.